content/docs/specification/content.md

```diff
@@ -320,3 +320,32 @@ The `proxy` type let's you proxy your requests to a specific upstream. It reads
 - Mandatory
 - Permitted values: "absolute URL"
 - Example: "to=https://example.com/subpage/"
+
+---
+
+# Wildcard Records
+
+A wildcard DNS record is a record in a DNS zone that will match requests for non-existent domain names. For example you can specifiy a record that matches all the subdomains that don't have a specific TXT record.
+
+Take a look at these sample records:
+
+```
+
+example.test.                    IN A   127.0.0.1
+_redirect.example.test.          IN TXT "v=txtv0;to=http://example.com;type=host;code=302"
+_redirect.specific.example.test. IN TXT "v=txtv0;to=http://specific.test;type=host;code=302"
+_redirect._.example.test.        IN TXT "v=txtv0;to=http://wildcard.test;type=host;code=302"
+
```

Comment:
Whitespace?

Reply:
All the whitespaces in snippets are because of the extra space bug. Just thought it would look better if they all were aligned.

Reply:
I understand the logic, but I feel this will be left, if the bug is fixed. Let's remove it and fix the bug in Syna faster. Or bug @mpourismaiel more :)

Reply:
Alright. Removed them and bugged Mahdi once again :smile: